chart/values.yaml

```diff
@@ -668,3 +668,9 @@ loki-distributed:
       aws:
         s3: "s3://access_key:secret_access_key@custom_endpoint/bucket_name"
         s3forcepathstyle: true  # set to 'false' to enable virtual-hosted-style URLs
+
+prometheus-openstack-exporter:
```

Comment:
We also want to disable the whole `prometheus-openstack-exporter` chart and not only the multicloud mode.